builder/make_log

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -eufo pipefail
+set -efo pipefail
```

Comment:
removing the set `-u` is not great (might result in uncaught errors down the line).... instead of relying on this, please do something like "${CI:-false}" to handle cases where `$CI` is not set.